File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        vector<int> res;

        for(int i=0;i<nums.size();i++){
            if(res.empty()){
                res.push_back(nums[i]);
            }
            else if(nums[i] != res.back()){
                res.push_back(nums[i]);
            }
        }
        nums = res;
        return nums.size();
    }
};

File: Data Structures & Algorithms/max-water-container/submission-0.py
class Solution:
    def maxArea(self, nums: List[int]) -> int:
        res = 0

        for i in range(len(nums)):
            for j in range(i+1, len(nums)):
                res = max(res, min(nums[i], nums[j]) * (j-i))
        return res